usage/run_project.md

```diff
@@ -17,13 +17,23 @@ Once the CLI is installed, help is available by typing `uo -h` from the command
 
     This creates a project folder containing the [example project](example.md), and downloads related weather files and detailed models to the appropriate folders.
 
-    Alternatively, create an empty base project folder by using: 
+    Create an empty base project folder by using:
+
     ```terminal
     uo -e -p <PROJECT_DIRECTORY_NAME>
     ```
     
     This creates project folder without an example FeatureFile and an empty weather folder. You can
     download weather files and add to this folder from energyplus.net/weather.
+
+    Overwrite an existing folder by usoing:
```

Comment:
@vtnate theres a typo - using
I added instructions for overwrite to develop https://github.com/urbanopt/urbanopt.github.io/blob/develop/usage/run_project.md